src/tests/utils.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy
 import pandas
-import yaml
+import numpy as np
```

Comment:
we have both `numpy` and alias `np` imported. 

Another thing is probably not a biggie, but sometimes we use `pandas` and sometimes we use the alias `pd`. It's not necessary to standardized, but just looks better.

Reply:
Ah yeah I can fix this

Reply:
I don't know that I would want to sweep the whole repo in this pull request to standardize pandas vs pd though.